cmd/api/init.go

```diff
@@ -182,6 +183,17 @@ func initEcho(cfg ApiConfig, db postgres.Storage, env string) *echo.Echo {
 			return nil
 		},
 	}))
+
+	timeout := 30 * time.Second
+	if cfg.RequestTimeout > 0 {
```

Comment:
mb e2e test on disconnection, so we could mimic the situation without catching it on dev?

Reply:
or is it even possible to do e2e test here? 🤔